packages/manager/apps/web/client/app/domain/zone/domain-zone-dns.html

```diff
@@ -28,6 +28,15 @@
                     data-translate="domain_tab_ZONE_no_zone_activate_btn"
                     data-ng-disabled="!ctrlDomainTabZoneDns.displayActivateZone"
                     data-ng-click="ctrlDomainTabZoneDns.activateZone()"
+                    data-ng-if="ctrlDomainTabZoneDns.region !== 'CA'"
+                ></button>
+                <button
+                    class="btn btn-block btn-default"
```

Comment:
```suggestion
                    class="oui-button ..."
```
https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-buttons-introduction--page

Reply:
thanks, i changed the dafault by oui-button of ui-kit.